Casks/f/feedflow.rb

```diff
@@ -0,0 +1,28 @@
+cask "feedflow" do
+  version "1.0.77"
+  sha256 :no_check
+
+  url "https://github.com/prof18/feed-flow/releases/download/#{version}-desktop/FeedFlow-#{version}.dmg",
+      verified: "github.com/prof18/feed-flow/"
+  name "FeedFlow"
+  desc "Minimalistic RSS Reader"
+  homepage "https://www.feedflow.dev/"
+
+  livecheck do
+    url :stable
+    regex(/^v?(\d+(?:\.\d+)+)[._-]desktop$/i)
+    strategy :github_latest do |json, regex|
+      match = json["tag_name"]&.match(regex)
+      next if match.blank?
+
+      match[1]
+    end
```

Comment:
```suggestion
    url :url
    strategy :github_latest
```

Will non-desktop releases be published in the GitHub repository, or will they continue to be tagged? If it's the latter, the suggestion above should suffice.

Reply:
Thanks!
There will be other tags but there will be only GitHub releases for desktop. So I guess the suggestion is fine?

Reply:
For now, there are only desktop releases, but I'm planning to start adding releases for other platforms!

Reply:
Then this suggestion should still be fine, granted you are manually setting the latest desktop release as `latest`.

Reply:
Ah perfect, thanks!